rust_src/src/buffers.rs

```diff
@@ -924,6 +924,18 @@ impl From<LispBufferOrCurrent> for LispBufferRef {
     }
 }
 
+/// Return false.
+/// If the optional arg BUFFER is provided and not nil, enable undoes in that
+/// buffer, otherwise run on the current buffer.
+#[lisp_fn(min = "0")]
+pub fn buffer_enable_undo(buffer: LispBufferOrCurrent) -> LispObject {
+    let mut buf: LispBufferRef = buffer.into();
+    if buf.undo_list_ == Qt {
+        buf.undo_list_ = Qnil;
+    }
+    Qnil
```

Comment:
When a DEFUN unconditionally returns `Qnil` the Rust version can be defined without a return value. Equivalent to `void func(....) { }` in C.